mpi-proxy-split/record-replay.h

```diff
@@ -740,7 +758,11 @@ namespace dmtcp_mpi
       }
 
       int datatype_decRef(MPI_Datatype datatype) {
-        return --_datatypeMap[datatype];
+	if (_datatypeMap.find(datatype) == _datatypeMap.end()) {
+          return -1;
+	} else {
+          return --_datatypeMap[datatype];
+	}
       }
 
       void datatype_get_stale_types(MPI_Datatype type,
```

Comment:
This function finds all stale types and adds them to the output parameter `staleTypes`. I change its name to `datatype_find_stale_types`. I will keep it to return void since nobody uses its return value.